iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/common/header/ColumnHeaderConstant.java

```diff
@@ -49,6 +49,8 @@ private ColumnHeaderConstant() {
   public static final String DEADBAND = "Deadband";
   public static final String DEADBAND_PARAMETERS = "DeadbandParameters";
   public static final String IS_ALIGNED = "IsAligned";
+  public static final String Template = "Template";
```

Comment:
Keep the code style consistent with the original code.
```suggestion
  public static final String TEMPLATE = "Template";
```

Reply:
Fixed.